src/governor/GovernorCoreFacet.sol

```diff
@@ -96,6 +96,15 @@ contract GovernorCoreFacet is AccessControl, IEIP712, IGovernor {
         return GovernorStorage.isProposalTokenEnabled(proposalToken);
     }
 
+    /**
+     * @notice determine if a counting strategy is valid to configure a proposal for voting with.
+     * @param countingStrategy the address of the counting strategy to check
+     * @return true if the counting strategy is a proposal token, false otherwise
+     */
+    function isCountingStrategyEnabled(bytes4 countingStrategy) public view returns (bool) {
```

Comment:
The doc says the "address" of the counting strategy but the param is a bytes4 so not an address. Can you help me understand?

Reply:
doc's wrong :D

Reply:
fixed